index.html

```diff
@@ -25,8 +28,8 @@ <h1>I'm a modal window 😍</h1>
         mollit anim id est laborum.
       </p>
     </div>
-    <div class="overlay hidden"></div>
+    <div class="overlay hidden">
 
-    <script src="/Users/admin/Downloads/complete-javascript-course-master/06-Modal/final/JS.js"></script>
+      <script src="/Users/admin/Downloads/complete-javascript-course-master/06-Modal/final/JS.js"></script>
```

Comment:
use the relative path 
```suggestion
      <script src="./JS.js"></script>
```